content/en/docs/Architecture/meta.md

```diff
@@ -21,9 +21,14 @@ Every component is extensible and modular such as it can be customized and repla
 
 In order to convert a Linux Distribution to Kairos, the distribution must meet the following requirements:
 
-- Either use `systemd` or another init system
-- Use `grub` as bootloader
-- If the system supposedly is meant to be used with EFI, the kernel needs to have enabled the `CONFIG_EFI_STUB` option ( see: https://docs.kernel.org/admin-guide/efi-stub.html)
+- Trusted Boot Images
+  - Use a recent enough version of `systemd` as init system
+  - Use `systemd-boot` as bootloader
+- Secure Boot Only Images
```

Comment:
How does one build such an image? Do we have a flow documented that allows building and image that is signed but it's not the full blown Trusted boot one?

Reply:
thats just a normal image. They all got secureboot enabled artifacts by default with the shim and such :D

Reply:
So they are signed with whatever key signs the shim? That changes per distro I guess?

Reply:
Yes indeed. They just work like a normal upstream iso